src/client/src/exporters/db.rs

```diff
@@ -163,88 +74,89 @@ impl AuroraClient {
+                .push_bind(event.instrumentation_type)
+                .push_bind(event.environment)
+                .push_bind(event.pipeline_type)
+                .push_bind(event.user_operator)
+                .push_bind(event.organization_id)
+                .push_bind(event.department)
                 .push_bind(event.run_id)
+                .push_bind(event.run_name)
                 .push_bind(event.pipeline_name)
-                .push_bind(event.nextflow_session_uuid)
-                .push_bind(event.job_ids)
-                .push_bind(event.tags_json)
-                .push_bind(event.event_timestamp.naive_utc())
+                .push_bind(event.job_id)
+                .push_bind(event.parent_job_id)
+                .push_bind(event.child_job_ids)
+                .push_bind(event.workflow_engine)
                 .push_bind(event.ec2_cost_per_hour)
                 .push_bind(event.cpu_usage)
                 .push_bind(event.mem_used)
                 .push_bind(event.processed_dataset)
-                .push_bind(event.process_status);
+                .push_bind(event.process_status)
+                .push_bind(event.attributes)
+                .push_bind(event.resource_attributes)
+                .push_bind(event.tags);
         }
         // there's an alternative, much more efficient way to push values
         // https://github.com/launchbadge/sqlx/blob/main/FAQ.md#how-can-i-bind-an-array-to-a-values-clause-how-can-i-do-bulk-inserts
         // however, unnest builds a tmp table from the array - and we're passing job_ids. Unnesting Arrays inside the arrays are tricky.
         // see https://github.com/launchbadge/sqlx/issues/1945
 
         info!(
-            "Inserting row for run_name: {}, pipeline_name: {}",
-            run_name, pipeline_name
+            "Inserting row for run_name: {}, pipeline_name: {}, run_id: {}",
+            run_name, pipeline_name, run_id
         );
 
         let mut builder = QueryBuilder::new(QUERY);
 
-        let mut data: Vec<_> = data
-            .into_iter()
-            .map(|e| {
-                EventInsert::try_new(
-                    e,
-                    run_name.to_string(),
-                    run_id.to_string(),
-                    pipeline_name.to_string(),
-                    e.process_status.as_str().to_string(),
-                )
-            })
-            .collect::<Result<Vec<_>>>()?;
+        let mut data: Vec<_> = logs.into_iter().map(EventInsert::from).collect();
 
         let rows_affected = match data.len() {
             0 => {
                 debug!("No data to insert");
                 return Ok(());
             }
-
             x if x * PARAMS >= BIND_LIMIT => {
-                debug!("Using transaction to insert data");
-                let mut transaction = self.pool.begin().await?;
-
+                debug!("Chunked insert with transaction due to bind limit");
+                let mut tx = self.pool.begin().await?;
```

Comment:
not to do now, but let's try to keep the variable names readable like `transaction` instead of `tx`